array_funcs.h

```diff
@@ -0,0 +1,77 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int get_array_size()
+{
+    int size;
+    do {
+        printf("Enter the size of the array: ");
+        scanf("%d", &size);
+        if (size <= 0) 
+        {
+            printf("Error: Size must be greater than zero\n");
+        }
+    } 
+    while (size <= 0);
+    return size;
+}
+
+int* get_array_elements(int size)
+{
+    int *nums_ptr = (int*)malloc(size * sizeof(int));
+    memset(nums_ptr, 0, size * sizeof(int));
+
+    for (int i = 0; i < size; i++)
+    {
+        printf("Enter the value of element %d: ", i + 1);
+        scanf("%d", &nums_ptr[i]);
+    }
+
+    return nums_ptr;
+}
+
```

Comment:
Add another function like this:
`int* create_new_array_with_random_elements (int size)`

this function should fill up the array with random integers. this will save you time from inputting elements every time when you run the program.